test/v1/connectors/CreateForeignPropertyConnectorSpec.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import mocks.MockAppConfig
+import uk.gov.hmrc.domain.Nino
+import v1.mocks.MockHttpClient
+import v1.models.outcomes.ResponseWrapper
+import v1.models.request.createForeignProperty._
+import v1.models.response.create.CreateForeignPropertyResponse
+
+import scala.concurrent.Future
+
+class CreateForeignPropertyConnectorSpec extends ConnectorSpec {
+
+  val nino = Nino("AA123456A")
+  val businessId = "XAIS12345678910"
+  val body = CreateForeignPropertyRequestBody(
+    "2020-01-01",
+    "2020-01-31",
+    Some(ForeignFhlEea(
+      ForeignFhlEeaIncome(5000.99, Some(5000.99)),
+      Some(ForeignFhlEeaExpenditure(
+        Some(5000.99),
+        Some(5000.99),
+        Some(5000.99),
+        Some(5000.99),
+        Some(5000.99),
+        Some(5000.99),
+        Some(5000.99),
+        Some(5000.99)
```

Comment:
Though the connector spec would work overall, these current values in the model are invalid as it contains a consolidateExpenses value